calico_containers/pycalico/datastore.py

```diff
@@ -56,9 +65,12 @@
 cali123456789ab.
 """
 
-# The default node AS number
+# The default node AS number.  Note that this value is also hard-coded into
```

Comment:
I did - but you still have to allow for the situation that ensure_global_config_exists() has not yet been called ...